taskRequests/README.md

```diff
@@ -3,13 +3,15 @@
 Firestore Task Requests collection data model
 
 ```
-id: <id>,
-taskId: <id>,
-title: string,
-requestedBy:  [string],
-purpose: string,
-type: string,
-isNoteWorthy: boolean,
-currentStatus: string,
-approvedTo?: string,
+{
+  approvedTo?: string,
+  id: string,
+  isNoteworthy: boolean,
+  priority: "LOW" | "Normal" | "HIGH",
+  purpose: string,
+  requestedBy: string[],
+  status: "APPROVED" | "WAITING",
```

Comment:
I believed Hhre `status` means creating a task, waiting for a task to be assigned, and assigning a task?

If yes then no change is required.

If not then I guess here `status` need a few more status state like **FINISHED**, and **CLOSED** ...etc

Correct me if I am wrong!!